quoridor-app/app/component/pawn.js

```diff
@@ -8,21 +9,27 @@ const styles = StyleSheet.create({
   }
 });
 
-export class Pawn extends React.Component {
-  render() {
-    return (
-      <Text
-        style={[
-          styles.pawn,
-          {
-            color: this.props.color,
-            top: this.props.position.top,
-            left: this.props.position.left
-          }
-        ]}
-      >
-        &#9823;
-      </Text>
-    );
-  }
-}
+const Pawn = ({ color, position }) => {
+  return (
+    <Text
+      style={[
+        styles.pawn,
+        {
+          color: color,
+          top: position.top,
+          left: position.left
+        }
+      ]}
+    >
+      &#9823;
+    </Text>
+  );
+};
+
+Pawn.prototype = {
+  color: PropTypes.string.isRequired,
+  top: PropTypes.number.isRequired,
```

Comment:
I didn't see this props in the component

Reply:
nope, it's a `position` props.